Makefile

```diff
@@ -0,0 +1,65 @@
+ENV ?= local
+KERNEL := $$(uname | tr '[:upper:]' '[:lower:]')
+GOPKGS := $$(go list ./... | grep -v /vendor/)
+VERSION ?= latest
+KUBECTL_VERSION = "v1.11.7"
```

Comment:
Since we are upgrading Kubernetes to 1.13.9 today and we are not going to apply this in production today, I think we can update everything to the new Kubernetes version :) What do you think?